### ISSUE TITLE
Understanding Vale.Spelling errors

### ISSUE THREAD

**Issue by pt1961** [jdkato/txtlint#527]
With a `.vale.ini`:

```ini
MinAlertLevel = suggestion

[*.md]
BasedOnStyles = Vale
```

I get some inconsistent `Vale.Spelling` errors with `file.md` that includes the contents:

```md
aB
aBb
aaBb
aaBbC
```

```
file.md
 3:1  error  Did you really mean 'aaBb'?  Vale.Spelling

✖ 1 error, 0 warnings and 0 suggestions in 1 file.
```

Why is `aaBb` reported as a spelling error, but the others are not? In particular, why is `aaBbC` not a spelling error?

How can I debug `Vale.Spelling` errors?

**Comment by jdkato** [comment]
To debug, you can turn off all default filters:

```yaml
extends: spelling
message: "Did you really mean '%s'?"
custom: true # no filters
level: error
```
and look at the [built-in dictionary](https://github.com/errata-ai/en_US-web).
